### ISSUE TITLE
Why is the accuracy is higher than reported in the paper?

### ISSUE THREAD

**Issue by erayyildiz** [onurgu/neural-turkish-morphological-disambiguator#2]
The printed accuracy is higher than the value reported in Shen et. al.'s paper.

Probably this is caused by the padding. While preparing the data to pass the model, numpy zero arrays with length `max_sentence_length`  are created and some of the values in input array are all zero because most of the sentences are shorter tham `max_sentence_length` parameter. 

Writing a custom accuracy metric instead of using keras default may be a solution.

By the way thanks for the implementation and sharing. It makes a lot of work for me.

**Comment by erayyildiz** [comment]
That was my fault. I missed to run the evaluation of your script. Sorry.

**Comment by onurgu** [comment]
No problem, you're welcome!

For those who are interested, you can run the evaluation script with this command:

```
python train.py --command predict --train_filepath data/train.merge.utf8 --test_filepath data/test.merge.utf8 --model_path ./models/ntd-nmd-20170619-06.epoch-32-val_acc-0.99507.hdf5 --run_name testing
```
